gravitee-apim-console-webui/src/management/configuration/notifications/notifications-list-settings.component.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, OnInit } from "@angular/core";
+import { combineLatest, Subject } from "rxjs";
+import { takeUntil, tap } from "rxjs/operators";
+
+import { NotificationSettingsNewService } from "../../../services-ngx/notification-settings-new.service";
+import {
+  NotificationsServices
+} from "../../../components/notifications/notifications-list/notifications-list-reusable.component";
+import { NotificationSettings } from "../../../entities/notification/notificationSettings";
+import { Notifier } from "../../../entities/notification/notifier";
+
+@Component({
+  selector: "notifications-list-settings",
+  template: require("./notifications-list-settings.component.html"),
+  styles: [require("./notifications-list-settings.component.scss")]
+})
+export class NotificationsListSettingsComponent implements OnInit {
```

Comment:
```suggestion
export class EnvrionementNotificationsListSettingsComponent implements OnInit {
```
I think it's easier to understand the scope of the component, WDYT ?

Reply:
done